### PR TITLE
feat(postSetup): Unit Test

### PR DESCRIPTION
**Pull request by lenaChenv**

### CODE REVIEW

File: packages/tees/src/lifecycle/postSetup.js [hunk 1]
```diff
@@ -209,7 +270,7 @@ function testCase(caseParams, fn, isOnly = false) {
           if (isSkipped) {
             break;
           }
-          execCase({
+          global.execCase({
```
[the remark this answers is not in the document]
Comment (summergan) [reply packages/tees/src/lifecycle/postSetup.js]: When I execute the test, there is an exception.
error info：
    TypeError: global.execCase is not a function

File: packages/tees/src/lifecycle/postSetup.js [hunk 2]
```diff
@@ -244,7 +299,7 @@ function testSkip(...args) {
 }
 
 function testOnly(...args) {
-  return testCase(...args, true);
+  return global.testCase(...args, true);
```
Comment (summergan) [inline packages/tees/src/lifecycle/postSetup.js]: Are you test？

File: packages/tees/src/lifecycle/postSetup.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (lenaChenv) [reply packages/tees/src/lifecycle/postSetup.js]: resolved

> When I execute the test, there is an exception.
> error info：
> TypeError: global.execCase is not a function

resolved

[the remark this answers is not in the document]
Comment (lenaChenv) [reply packages/tees/src/lifecycle/postSetup.js]: resolved

> When I execute the test, there is an exception.
> error info：
> TypeError: global.execCase is not a function

resolved

File: packages/tees/src/lifecycle/postSetup.js (hunk 2 above)
Comment (lenaChenv) [reply packages/tees/src/lifecycle/postSetup.js]: resolved

**Pull request closed without merging**